src/helper/selection-tools/reshape-tool.js

```diff
@@ -33,7 +33,14 @@ const ReshapeModes = keyMirror({
 class ReshapeTool extends paper.Tool {
     /** Distance within which mouse is considered to be hitting an item */
     static get TOLERANCE () {
-        return 4;
```

Comment:
I think it would make more sense to define this as a HANDLE_SIZE of 8 with a TOLERANCE of 2.25. I think tolerance is usually defined as the space outside of the visible thing you're clicking which still counts as a click on the object.